client/my-sites/email/email-providers-comparison/index.jsx

```diff
@@ -271,13 +280,17 @@ class EmailProvidersComparison extends React.Component {
 					// Stay on the page to show the relevant error(s)
 					return;
 				}
-				this.isMounted && page( '/checkout/' + selectedSiteSlug );
+				this.isMounted && page( '/checkout/' + selectedSite.slug );
 			} );
 	};
 
 	renderEmailForwardingCard() {
 		const { domain, translate } = this.props;
 
+		if ( this.isUpgrading( domain ) ) {
```

Comment:
This can probably be the first statement as `isUpgrading()` doesn't actually take any arguments.
```suggestion
		if ( this.isUpgrading() ) {
```

Reply:
Good catch!